### PR TITLE
[T4A4][F10-B1] Calvin Yang JiaWei

### PR DESCRIPTION
**Pull request by origiri**

### CODE REVIEW

**Comment by origiri** [comment]
Ready for review

File: src/seedu/addressbook/data/person/Printable.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+    default String getPrintableString(Printable... printables) {
+        StringBuilder printableString = new StringBuilder();
+        for(Printable printable: printables) {
+            printableString.append(printable.getPrintableString() + ", ");
+        }
+        return printableString.substring(PRINTABLE_STRING_START, printableString.length() - PRINTABLE_STRING_EXCESS);
```
Comment (Candiie) [inline src/seedu/addressbook/data/person/Printable.java:18]: Cool implementation to remove the ", " at the end XD ! Do take a look @ StringJoiner or String.join that was mentioned in tutorial :) ~ trust me it'll make ur life easier !

**Comment by Candiie** [comment]
Some comments added. Please close the PR after reading comments.

Hi Hi, great effort! Do take a look @ the comment above ~

File: src/seedu/addressbook/data/person/Printable.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
```
Comment (Candiie) [inline src/seedu/addressbook/data/person/Printable.java:3]: Hi Hi, you are missing header comment here. Remember, all non trivial classes/methods should have javadoc format header comments.

**Pull request closed without merging**